web/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, user-scalable=no">
     <title>Maunium sticker picker</title>
-
-    <link rel="modulepreload" href="src/widget-api.js"/>
+    <script src="https://unpkg.com/matrix-widget-api@1.6.0/dist/api.js"></script>
+    <script type="module" src="src/widget-api.js"></script>
```

Comment:
This change also looks wrong

Reply:
This seemed necessary at the time to get matrix-widget-api to load, but it was unnecessary. In any case, it's no longer required now, reverted.